src/libs/colorpicker.c

```diff
@@ -175,7 +175,10 @@ static void _update_picker_output(dt_lib_module_t *self)
     switch(input_color)
     {
       case 0: // rgb
-        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d",
+        snprintf(tooltip, sizeof(tooltip), "%3d   %3d   %3d   %02X%02X%02X",
```

Comment:
Can we add some kind of separator here ? Maybe "%3d   %3d   %3d  - %02X%02X%02X" or using | ?

Or maybe "hex: " to clearly indicate that the values are hex encoded ?

Reply:
Yeah that is better. Changed to `0   128   255  (0x0080FF)`